neil

```diff
@@ -338,10 +396,17 @@ add
       "build" (add-build opts)
       "kaocha" (add-kaocha opts))))
 
+(defn dep [[subcommand & opts]]
+  (let [opts (parse-opts opts)
+        opts (with-default-deps-edn opts)]
```

Comment:
Love how you're working with options. Great fit for Clojure, great fit for Unix CLIs.